### ISSUE TITLE
Update maintenance section after corresponding web updates

### ISSUE THREAD

**Issue by kafitzgerald** [NCAR/geocat-hugo#10]
A reminder to update the maintenance section on this site after updating the corresponding package websites.

**Comment by jukent** [comment]
https://ncar.github.io/geocat-hugo/software/

I believe we're using the correct websites? We do want to improve/update the maintenance blurb.

**Comment by kafitzgerald** [comment]
Yeah, I think we may want to update the blurb a bit and potentially link to some more recent NCL transition updates.  This is probably more of a polish thing though.
